content/chainguard/chainguard-images/how-to-use/images-directory/index.md

```diff
@@ -1,15 +1,13 @@
 ---
-title: "Using the Chainguard Images Directory"
+title: "Using the Chainguard Directory and Console"
 linktitle: "Images Directory"
```

Comment:
I (and @angela-zhang) think the linktitle should be either "Browsing Images" or "Using the Chainguard Directory and Console". Personally, I prefer the former just because it's shorter :smile:

Reply:
I discussed with Angela that this would be done when we do the batch renaming, thanks!